src/Ng/CommandHelpers.cs

```diff
@@ -428,21 +428,11 @@ public static Lucene.Net.Store.Directory GetCopyDestLuceneDirectory(IDictionary<
             return handlerFunc;
         }
 
-        public static IEnumerable<EndpointFactory.Input> GetEndpointFactoryInputs(IDictionary<string, string> arguments)
+        public static EndpointConfiguration GetEndpointConfiguration(IDictionary<string, string> arguments)
         {
-            return arguments.GetOrThrow<string>(Arguments.EndpointsToTest).Split(';').Select(e =>
-            {
-                var endpointParts = e.Split('|');
-
-                if (endpointParts.Count() < 2)
-                {
-                    throw new ArgumentException(
-                        $"\"{e}\" is not a valid endpoint! Each endpoint must follow this format: " +
-                        $"<endpoint-to-test-name>|<endpoint-to-test-cursor>.");
-                }
-
-                return new EndpointFactory.Input(endpointParts[0], new Uri(endpointParts[1]));
-            });
+            var registrationCursorUri = arguments.GetOrDefault<Uri>(Arguments.RegistrationCursorUri);
```

Comment:
>GetOrDefault [](start = 50, length = 12)

Should we require these arguments? I don't think there's ever a case we'll want to to monitor registration but not the flat container.